backend/src/sms/middlewares/sms-callback.middleware.ts

```diff
@@ -48,7 +65,26 @@ const parseEvent = async (
   }
 }
 
+const parseTransactionalEvent = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+): Promise<Response | void> => {
+  try {
+    await SmsCallbackService.parseTransactionalEvent(req)
+    logger.info({
+      message: 'Successfully parsed transactional event',
```

Comment:
I dont think we need to log at this point of success, if any, we should log before the processing/parsing

Reply:
moved